Testscripts/Linux/utils.sh

```diff
@@ -3559,4 +3559,35 @@ function Run_SSHCommand()
 			ssh "${ip}" "${cmd}"
 		fi
 	done
+}
+
+function Check_lsvmbus()
+{
+	# check if lsvmbus exists, or the running kernel does not match installed version of linux-tools
+	lsvmbus_path=$(which lsvmbus)
```

Comment:
Try using ```command -v``` for detecting whether utility is present or not. It is built-in in shell.